find_target_name_from_header.py

```diff
@@ -99,7 +122,7 @@ def query_simbad(date,coords,name=None,limit_G_mag=15,metadata=None,force_cm=Fal
     customSimbad = Simbad()
     customSimbad.add_votable_fields('flux(U)','flux(B)','flux(V)','flux(R)',\
                                     'flux(I)','flux(G)','flux(J)','flux(H)',\
-                                    'flux(K)','id(HD)','sp','otype','otype(V)','otype(3)',\
```

Comment:
Last comment, I'd recommend to keep the K band filter, as this is a useful information, especially in the context of SPHERE+ where we want to guide on fainter targets. Knowing their K mag is important

Reply:
(This is just for the test whether a star is bright enough or not for AO that we don't want to use the K band, later on in the code, sorry if that was not clear on my email)

Reply:
Okay! I've reinserted it into the code line